### PR TITLE
Christy manifest history

### PR DESCRIPTION
**Pull request by tophj-ibm**

### CODE REVIEW

**Comment by tophj-ibm** [comment]
that's a lot of commits

**Comment by tophj-ibm** [comment]
closing because that rebase... I don't even know

File: cli/command/manifest/inspect_test.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+// use testdata and golden for this stuff
+func TestNewInspectCreateSuccess(t *testing.T) {
+	testCases := []struct{
+		name		string
+		args		[]string
+	}{
+		{
+			name:	"simple",
+			args:	[]string{"busybox"},
```
Comment (tophj-ibm) [inline cli/command/manifest/inspect_test.go:45]: @dnephin cli test question, this is part of testing for https://github.com/docker/cli/pull/138

Is it okay to use an actual image name here? I'm looking at the cli push/pull tests and I see that they are all using "image:tag" to test things, but I can't figure out how they get past the fact that the image "image:tag" doesn't exist.

Comment (dnephin) [reply cli/command/manifest/inspect_test.go:45]: Yup, you can use anything here. These are all unit tests, so they never hit a real API. The client is a fake that returns whatever response you want it to return.

**Pull request closed without merging**